tests/integration/test_charm.py

```diff
@@ -18,10 +23,66 @@ async def test_build_and_deploy(ops_test: OpsTest):
     image_path = METADATA["resources"]["oci-image"]["upstream-source"]
     resources = {"oci-image": image_path}
 
+    await ops_test.run(
+        "kubectl",
+        "label",
+        "namespace",
+        "default",
+        "serving.kubeflow.org/inferenceservice=enabled",
+        "--overwrite",
+        check=True,
+    )
+
```

Comment:
Can we change this to use lightkube?  I don't know if there's a convenience function that updates in place like this, but could always get the namespace resource, add your updates to the labels, then submit the resulting labels as a patch